README.md

```diff
@@ -130,6 +130,8 @@ The master branch and open pull requests will be formatted automatically by a Gi
 
 You need to enable *local and third-party actions* in the Actions tab of your fork's settings to enable automatic formatting. If you disable actions, no formatting will take place on your branches.
 
+It is strongly recommened that you [create a Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) with permissions to write to your repos and trigger workflows. [Store this token as a secret](https://docs.github.com/en/free-pro-team@latest/github/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) named `WIDELANDS_FORMAT_TOKEN` in your fork. If you do not do this, our continuous integration suite will not work properly for branches in your fork.
```

Comment:
```suggestion
It is strongly recommened that you [create a Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) with permissions to write to your repos and trigger workflows. [Store this token as a secret](https://docs.github.com/en/free-pro-team@latest/github/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) named `WIDELANDS_FORMAT_TOKEN` in your fork. If you do not do this, our continuous integration suite will not work properly for branches in your fork. Use the following token settings:
* Note: `WIDELANDS_FORMAT_TOKEN`
* Select scopes: Repo -> `public_repo`
```

Reply:
The token's Note does not matter, `WIDELANDS_FORMAT_TOKEN` only refers to the secret's name.

Reply:
I clearly didn't understand the documentation then. Please make the instructions simple stupid with bullet points. Basic structure of the instructions should be like this:

Step 1: Create a token in your **profile** and get the hash code

Step 2: Create a secret in your **fork**, name it `WIDELANDS_FORMAT_TOKEN` and paste the hash code

I think this change would be good to have, but we don't want to make creating pull requests too difficult for new contributors.

Reply:
> The token's Note does not matter, `WIDELANDS_FORMAT_TOKEN` only refers to the secret's name.

I ran through the process. The tokens note/name is a required field and i had no idea how i should name it. An example would had helped me a lot.

All other steps are clear to me now :-) I had to carefully search some things though.

Reply:
If it doesn't matter, what does it matter ;) The purpose of this Note field is just that when you look at the list of your tokens you will notice if someone hacked your account and added a token you do not recognize. Just call it `Hello World` or `Foo Bar Baz` or `This is a super-important token for a super-important purpose` or `Widelands formatting bot authentification` …